aws/main.tf

```diff
@@ -157,14 +163,31 @@ module "edge" {
   master_ip = "${element(module.master.local_ip_v4, 0)}"
 }
 
+#
```

Comment:
Please use single # for comments: `# The code below (from here to end) should be identical for all cloud providers`